spec/lucky/select_helpers_spec.cr

```diff
@@ -21,20 +36,22 @@ end
 
 describe Lucky::SelectHelpers do
   it "renders select" do
-    (view.select_input form.company_id do; end).to_s.should contain <<-HTML
+    view.select(form.company_id).html.to_s.should contain <<-HTML
     <select name="company:company_id"></select>
     HTML
   end
 
-  it "renders option" do
-    (view.options_for_select form.company_id, [{"Volvo", 2}]).to_s.should contain <<-HTML
-    <option value="2" selected="false">Volvo</option>
+  it "renders options" do
+    view.options(form.company_id, [{"Volvo", 2}, {"BMW", 3}])
+      .html.to_s.should contain <<-HTML
+    <option value="2">Volvo</option><option value="3">BMW</option>
     HTML
   end
 
   it "renders selected option" do
-    (view.options_for_select form.company_id, [{"Volvo", 1}]).to_s.should contain <<-HTML
-    <option value="1" selected="true">Volvo</option>
+    view.options(form.company_id, [{"Volvo", 1}, {"BMW", 3}])
+      .html.to_s.should contain <<-HTML
+    <option value="1" selected="true">Volvo</option><option value="3">BMW</option>
```

Comment:
I think this should also have another assertion that checks that `BMW` does not contain a `selected` attribute

Reply:
That should be covered by asserting the `<option value="3">BMW</option>` shouldn't it? Nevertheless I changed the assertions from `containt` to `eq` as I dont think we want anything else in there anyway.

Reply:
Oh you're right. I didn't see that last option. This looks great. And I like the change to `eq` 👍